README.md

```diff
@@ -1,55 +1,33 @@
-![logo](./logo.png)
+# Safient Core SDK
 
-Secure crypto secret exchange and inheritance protocol
+JavaScript SDK to manage and interact with the safes on Safient protocol.
 
-## About
-Digital crypto assets are the talk of the town and gaining traction rapidly among all kinds of individuals and organizations. Due to the higher traction, these assets are highly valuable. So, one has to make sure that their assets are safe and sound.
-
-Owning and securing the digital crypto-asset unlike other digital assets can be challenging due to the self-sovereign nature of the ownership. It means that the owner is the only safe keeper of the assets. For many users, this might be an intimidating role as they don't want to completely responsible for preserving the assets against attacks and natural disasters. Every user always desires a trusted recovery method in case of tragedies. But there is no standardized way to easily and securely back up and recover the assets. 
-
-Safex promises to securely and conveniently store any critical information that is needed to access and recover the assets in case of any tragic events. Safex also provides a trustless yet safe way to transfer and inherit the assets by close ones whenever such an unfortunate scenario occurs.
-
-Safex uses decentralized storage, smart contracts, and cryptography techniques to build a protocol that helps to store, recover, inherit crypto assets and other confidential information. In essence, you can think of Safex as your trusted guardian even though it is completely trustless.
-
-
-## Technologies used:
-* [Ceramic IDX](https://idx.xyz/)
-* [Textile ThreadDb](https://docs.textile.io/threads/)
+## Installation
 
+```bash
+  git clone https://github.com/safient/safient-core.git
+  cd safient-core
+  npm install
+```
 
-## Getting Started
+## Running Tests
 
-First, run the middleware server:
+Terminal 1
 
```

Comment:
Add the instruction to create an `.env` file with KEYS, SECRET and DB_FILE_NAME.